ipmininet/router/config/base.py

```diff
@@ -2,6 +2,10 @@
 that is able to provide configurations for a set of routing daemons.
 It also defines the base class for a routing daemon, as well as a minimalistic
 configuration for a router."""
+from builtins import map
+from future.utils import iteritems
+from six import string_types
```

Comment:
Not sure if pulling in six is really needed when all we want could be adding in the baselevel __init__.py=
```python
try:
  basestring
except NameError:
  basestring = str
```